index.html

```diff
@@ -1089,6 +1111,13 @@ <h3>Pinebook Pro</h3>
 							<p>RK3399 1.8GHz | 6 cores | ARMv8</p>
 						</a>
 					</div>
+					<div class="col-xl-2 col-lg-3 col-md-4 col-6 mix radxa">
+						<a href="#downloadinfo" class="thumbnail show_hide" rel="#radxa">
+							<img src="images/sbc_images/radxazero.jpg" alt="Radxa Zero photo" width="8" height="5" loading="lazy">
+							<h3>Radxa Zero</h3>
+							<p>Amlogic S905Y2 1.8GHz | 4 cores | ARMv8</p>
+						</a>
+					</div>
```

Comment:
Would be good to move this below ROCK Pi S, so all Radxa tiles are shown together.

Reply:
maybe or not
ROCK Pi S based on Rockchip 
Radxa Zero on Amlogic
I do not know which is more correct

Reply:
We sort it by manufacturer, not by SoC, so this is to keep the sorting that is already present.

Reply:
![image](https://user-images.githubusercontent.com/16898685/142470004-1258649b-78b9-444b-8591-557edc4cdcee.png)
here?

Reply:
Yeah exactly, but also when `[All]` are shown, they should show up together, which is assured by by sorting those tile blocks accordingly.